src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -544,6 +544,8 @@ namespace winrt::TerminalApp::implementation
         // possible that the focus events won't propagate immediately. Updating
         // the focus here will give the same effect though.
         _UpdateActivePane(newPane);
+
+        return { original, newPane };

```

Comment:
Nothing uses the `newPane` tuple value. Do we need it?

Reply:
Not really, but I included it for parity with `Pane::Split`. Seemed to me like they should return the same thing